### PR TITLE
Author and listing level author profile error states

### PR DESCRIPTION
**Pull request by toboid**
[REPLAT-821](https://nidigitalsolutions.jira.com/browse/REPLAT-821)

Adds error states for errors getting author or articles, with ability to refetch

### CODE REVIEW

File: packages/author-profile/author-profile-content.js [hunk 1]
```diff
@@ -86,6 +93,27 @@ class AuthorProfileContent extends React.Component {
       />
     );
 
+    const renderAuthorHead = () => (
```
Comment (craigbilner) [inline packages/author-profile/author-profile-content.js]: minor: does this need to be a func?

File: packages/author-profile/author-profile-error.js [hunk 2]
```diff
@@ -1,11 +1,82 @@
...
+    marginBottom: 20
+  }
+});
+
+const AuthorProfileError = ({ refetch }) => (
+  <View style={styles.container}>
+    <View style={styles.imageContainer}>
+      <Image
+        uri="https://www.thetimes.co.uk/d/img/internal-error-c45d0e8347.png"
+        aspectRatio={0.9090909090909091}
```
Comment (craigbilner) [inline packages/author-profile/author-profile-error.js]: :-)

File: packages/author-profile/author-profile-error.js [hunk 3]
```diff
@@ -1,11 +1,82 @@
...
+      />
+    </View>
+    <View style={styles.body}>
+      <View style={styles.messagingContainer}>
+        <Text style={styles.heading}>Something&apos;s gone wrong</Text>
+        <Text style={styles.message}>
+          We cant load the page you have requested. Please check your network
```
Comment (craigbilner) [inline packages/author-profile/author-profile-error.js]: typo: can't

File: packages/author-profile/author-profile-error.web.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    max-width: 365px;
+  `
+});
+
+const AuthorProfileError = ({ refetch }) => (
+  <ErrorContainer>
+    <MessagingContainer>
+      <Heading>Something&apos;s gone wrong</Heading>
+      <Message>
+        We cant load the page you have requested. Please check your network
```
Comment (craigbilner) [inline packages/author-profile/author-profile-error.web.js]: typo: can't

File: packages/author-profile/author-profile-listing-error.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    marginRight: 5
+  }
+});
+
+const AuthorProfileListingError = ({ refetch }) => (
+  <View style={styles.body}>
+    <View style={styles.messagingContainer}>
+      <Text style={styles.heading}>Something&apos;s gone wrong</Text>
+      <Text style={styles.message}>
+        We cant load the page you have requested. Please check your network
```
Comment (craigbilner) [inline packages/author-profile/author-profile-listing-error.js]: typo: can't

File: packages/author-profile/author-profile-listing-error.web.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    width: 200px;
+    padding-bottom: 10px;
+  `
+});
+
+const AuthorProfileListingError = ({ refetch }) => (
+  <ListingError>
+    <Heading>Something&apos;s gone wrong</Heading>
+    <Message>
+      We cant load the page you have requested. Please check your network
```
Comment (craigbilner) [inline packages/author-profile/author-profile-listing-error.web.js]: typo: can't

File: packages/author-profile/author-profile-error.js (hunk 3 above)
Comment (toboid) [reply packages/author-profile/author-profile-error.js]: 😳  fixed

File: packages/author-profile/author-profile-error.js (hunk 2 above)
Comment (toboid) [reply packages/author-profile/author-profile-error.js]: changed to `700/770` for clarify

**Comment by craigbilner** [comment]
current issues:

- [ ] retry on failed page breaks paging
- [ ] paging after failure is broken

**Pull request merged**